torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1633,6 +1633,26 @@ def sample_repeat_tile(op_info, device, dtype, requires_grad, **kwargs):
 
     return samples
 
+def sample_unsqueeze(op_info, device, dtype, requires_grad, **kwargs):
+    shapes_and_axes = [
+        ((3, 4, 5), 0),
+        ((3, 4, 5), 1),
+        ((3, 4, 5), 3),
+        ((3, 4, 5), -1),
+        ((3, 4, 5), -3),
+        ((), 0)
+    ]
+
+    samples = []
+    for shape, axis in shapes_and_axes:
+        tensor = make_tensor(shape, device, dtype, low=None, high=None,
+                             requires_grad=requires_grad)
+
+        for t in (tensor, tensor.T):
```

Comment:
This part is a little tricky. Taking the transpose is clever, but it creates a dependency between two sample inputs.

For now I suggest simplifying and simply not taking the transpose and constructing a SampleInput from each tensor directly.

In the future we should automate more noncontiguous testing.

Reply:
@mruberry Fixed! Thanks!